src/popup/components/TransactionItem.vue

```diff
@@ -52,24 +52,22 @@
 </template>
```

Comment:
Is `toUpperCase()` still required in line 45?

Reply:
replaced with CSS style